isis/tests/data/defaultImage/defaultCube.pvl

```diff
@@ -89,6 +89,16 @@ Object = IsisCube
     Status   = Nominal
   End_Group
 End_Object
+Object = History
```

Comment:
I know why hyb2 needs these, but why does the default cube need these? They are not required objects.

Reply:
hyb2pds4gen needed these as it expect them to be there as part of the translation. Since it didn't seem like a good idea to change the functionality of `ProcessExportPds4` by making them optional, so I added them to the cube. I imagine all Pds4 related tests will have this issue so I made the changes to defaultcube fixture so someone else won't run into the same issue I did (also doesn't negatively impact other tests).